core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/SubmissionApiService.scala

```diff
@@ -44,6 +50,18 @@ trait SubmissionApiService extends UserInfoDirectives {
           }
         }
       } ~
+      path("workspaces" / Segment / Segment / "retrySubmission" / Segment) { (workspaceNamespace, workspaceName, submissionId) => // keeping all logic here and slowly moving out
```

Comment:
could this be `/api/workspaces/{workspaceNamespace}/{workspaceName}/submissions/{submissionId}/retry` instead?

Reply:
I'm game for this change, I'll leave this up for discussion for a little while longer so everyone can give their two cents. I like this change given the other endpoints with existing similar paths and the prerequisite nature of the new endpoint coinciding with the nesting.